docs/assignment1.md

```diff
@@ -1,62 +1,73 @@
+| F2   | Movements  | The player must move around the virtual environment using *move* followed by a direction: *North, East, West, South*. These command will move the player to the next virtual area. |
+| F3   | Narrator   | The narrator is a way for the system to interact with the user: Let the user know what is going on. The narrator will describe the *field of view* of the player at any given time when: Moving between rooms, the command info is received. We define *field of view* as what all objects and properties of the immediate area around the user. That is, the area the user is currently in. For example, if the player is in a room that is dark, there is only an entrace to the west, and in this area, there lies a fridge, broom and coffee machine. The game would narrate *"You are in a dark room. The the west you see an opening to another area. In this room, you see a fridge, broom and coffee machine"*.  <br>There are also other commands available, which will have a special naration. For example, when picking up and object, the narrator will reply with a simple confirmation, simply repeating the action. If the action fails, the system should reply accordingly. <br>The command *diagnose* will trigger the narrator to list the health of the player. For example, you are wounded on your left hand, 80% health remaining<br>The command *inventory* will list all of the contents, without any specific ordering, the player currently has in their posession. |
+| F4   | Game-info  | The player can get the statistics with the command *stats*. The statistics include, time being played, current score (see F7). Furthermore, information about the current gamestate can be achieved with the use of special commands. See F3 for more info on these. |
+| F5   | Map        | A list that keeps track of the areas already visited. Quite self explanatory. **Bonus:** Include an option to add a graphic for each item which can then be displayed as a backpack. |
```

Comment:
Bonus describing wrong graphic display. it does not make sense for a backpack to be displayed in the map